core/assets/bundles/bundle_tl.properties

```diff
@@ -88,18 +88,18 @@ committingchanges = Comitting Changes
 done = Tapos
 feature.unsupported = Ang iyong device ay hindi sinusuporta ang feature na'to.
 
-mods.alphainfo = Keep in mind that mods are in alpha, and[scarlet] may be very buggy[].\nReport any issues you find to the Mindustry GitHub or Discord.
+mods.alphainfo = Laging tandaan na ang mga mod ay nasa Alpha, at[scarlet] baka may mga bug p[].\nI-report ang mga problema sa Mindustry GitHub o sa Discord.
 mods.alpha = [accent](Alpha)
 mods = Mods
-mods.none = [LIGHT_GRAY]No mods found!
+mods.none = [LIGHT_GRAY]Walang mahanap na mod!
 mods.guide = Modding Guide
 mods.report = Report Bug
 mods.openfolder = Open Mod Folder
-mod.enabled = [lightgray]Enabled
-mod.disabled = [scarlet]Disabled
+mod.enabled = [lightgray]Nai-bukas
+mod.disabled = [scarlet]Nai-sara
 mod.disable = Disable
-mod.delete.error = Unable to delete mod. File may be in use.
-mod.requiresversion = [scarlet]Requires game version: [accent]{0}
+mod.delete.error = Hindu ma-delete ang mod. Ang file ay baka ginagamit pa.
```

Comment:
i was thinking of "Hindi ma-tangal ang mod."
my main language is Filipino/Tagalog, but i suck at it xD